api/v1alpha1/databasecluster_types.go

```diff
@@ -225,6 +228,26 @@ type Backup struct {
 	Enabled bool `json:"enabled"`
 	// Schedules is a list of backup schedules
 	Schedules []BackupSchedule `json:"schedules,omitempty"`
+	// PITR is the configuration of the point in time recovery
+	PITR PITRSpec `json:"pitr,omitempty"`
+}
+
+// PITRSpec represents a specification to configure point in time recovery for a database backup/restore.
+type PITRSpec struct {
+	// Enabled is a flag to enable PITR
+	Enabled bool `json:"enabled,omitempty"`
+	// StorageName is the name of the BackupStorage where the PITR is enabled
+	StorageName string `json:"storageName"`
+	// TimeBetweenUploadsSec number of seconds between the binlogs uploads
+	TimeBetweenUploadsSec *int `json:"timeBetweenUploadsSec,omitempty"`
+	// CompressionType type of compression (psmdb only, default s2)
+	// +kubebuilder:validation:Enum:=gzip;snappy;lz4;s2;pgzip;zstd
+	CompressionType compress.CompressionType `json:"compressionType,omitempty"`
+	// CompressionLevel level of compression (psmdb only, more info
+	// https://docs.percona.com/percona-backup-mongodb/reference/backup-options.html#backupcompressionlevel)
+	CompressionLevel *int `json:"compressionLevel,omitempty"`
+	// Resources the compute resource requirements (pxc only, default no limits applied)
+	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
Do we really need to expose these to the users right now or could we just set some sensible defaults ourselves in the controller?
We've been bitten by a complex API in the past which eventually lead to a big refactor a few months ago to remove a lot of options that most users didn't care about. These seem similar to the type of things we removed back then, I don't want us to make the same mistake of making the interface too bloated.

Reply:
Slava mentioned once he would add them, but I don't mind deleting it. Let's start with smth simple.
Done.